gtsfm/loader/astronet_loader.py

```diff
@@ -0,0 +1,235 @@
+"""Simple loader class that reads a dataset with metadata formatted in the COLMAP style.
+
+Authors: Travis Driver
+"""
+
+import os
+from pathlib import Path
+from typing import Optional, Dict, Tuple, List
+
+from gtsam import Cal3Bundler, Pose3, Rot3, Point3, SfmTrack
+
+import gtsfm.utils.io as io_utils
+import gtsfm.utils.logger as logger_utils
+from gtsfm.common.image import Image
+from gtsfm.loader.loader_base import LoaderBase
+
+from thirdparty.colmap.scripts.python.read_write_model import read_model
+from thirdparty.colmap.scripts.python.read_write_model import Camera as ColmapCamera
+from thirdparty.colmap.scripts.python.read_write_model import Image as ColmapImage
+from thirdparty.colmap.scripts.python.read_write_model import Point3D as ColmapPoint3D
+
+
+logger = logger_utils.get_logger()
+
+
+class AstroNetLoader(LoaderBase):
+    """Simple loader class that reads an AstroNet data segment.
+
+    Refs:
+    - https://github.com/travisdriver/astronet
+    """
+
+    def __init__(
+        self,
+        data_dir: str,
+        use_gt_intrinsics: bool = True,
```

Comment:
I think we aren't handling the case when `use_gt_intrinsics` would be false in the class. Would we expect a silent failure, or what sort of error would be raised?

My sense is that if the AstroNet data will always have intrinsics, we can just note this in the docstring, remove this arg, and throw a `RuntimeError` if the intrinsics files are not found on disk in the constructor.

Reply:
Yeah sorry, I thought `use_gt_intrinsics` was required by the `BaseLoader`. I'll remove it and throw a runtime error.

Reply:
Done